### PR TITLE
Update release notes for Kafka connector 3.1.1

### PR DESCRIPTION
**Pull request by avsej**

### CODE REVIEW

File: content/connectors/kafka-3.1/release-notes.dita [hunk 1]
```diff
@@ -5,7 +5,19 @@
     <shortdesc>Release notes for the 3.1 version of the Kafka Connector. </shortdesc>
     <conbody>
         <section>
-            <title>Couchbase Kafka Connector 3.1.0 GA (03 January 2016)</title>
+            <title>Couchbase Kafka Connector 3.1.1 GA (21 Fabruary 2017)</title>
+            <p>Version 3.1.1 is maintenance release. It contains fixes for resuming DCP streams after restart.</p>
+            <ul>
+                <li>
+                    <xref href="https://issues.couchbase.com/browse/KAFKAC-56" format="html" scope="external">KAFKAC-56</xref>:
+                    Session state might be left partially initialized, which leads to rolling back to zero sequence
```
Comment (mattcarabine) [inline content/connectors/kafka-3.1/release-notes.dita]: Could you change "zero sequence number" to "a sequence number of zero (0)".

**Comment by avsej** [comment]
@mattcarabine I fixed it, sorry I missed notification. Next time I think you can upload small edits like this directly to this feature branch. It is the same repo, not fork, so everyone has write access.

**Comment by mattcarabine** [comment]
@cb-docs-robot backport to 4.5

**Comment by mattcarabine** [comment]
@cb-docs-robot backport to 4.6

**Pull request merged**